### PR TITLE
added unit-tests

### PR DESCRIPTION
**Pull request by lilymammadova**

### CODE REVIEW

File: src/test/java/org/liliya/hotelapp/service/ApartmentServiceImplTest.java [hunk 1]
```diff
@@ -0,0 +1,173 @@
...
+
+public class ApartmentServiceImplTest {
+    private ApartmentService apartmentService;
+
+    @BeforeEach
+    void setUp() {
+        apartmentService = new ApartmentServiceImpl();
+    }
+
+    //Test for  method register
```
Comment (Alexmansar) [inline src/test/java/org/liliya/hotelapp/service/ApartmentServiceImplTest.java]: no comments needed

File: src/test/java/org/liliya/hotelapp/service/ApartmentServiceImplTest.java [hunk 2]
```diff
@@ -0,0 +1,173 @@
...
+
+        //Assert
+        assertEquals(1, apartments.size(), "Should be one apartment registered");
+        assertEquals(100, apartments.get(0)
+                .getPrice(), "The price of new registered apartment should be 100");
+        assertEquals(ReservationStatus.AVAILABLE, apartments.get(0)
+                .getReservationStatus(), "Reservation status of new registered apartment should be AVAILABLE");
+    }
+
+    //Test for method reserve
```
Comment (Alexmansar) [inline src/test/java/org/liliya/hotelapp/service/ApartmentServiceImplTest.java]: same in general

File: src/test/java/org/liliya/hotelapp/ui/ConsoleTest.java [hunk 3]
```diff
@@ -0,0 +1,147 @@
...
+    private ApartmentService apartmentService;
+    @Mock
+    private Scanner scanner;
+    @InjectMocks
+    private Console console;
+
+    @BeforeEach
+    void setUp() {
+        console = new Console(apartmentService, scanner);
+    }
```
Comment (Alexmansar) [inline src/test/java/org/liliya/hotelapp/ui/ConsoleTest.java]: try to remove it

**Review by Alexmansar — CHANGES_REQUESTED** [review]
@lilymammadova Hi) in general good job) 
Have general comments regarding comments
and what about CI?

**Comment by Alexmansar** [comment]
@lilymammadova 
Fix your pom xml
image

File: src/test/java/org/liliya/hotelapp/service/ApartmentServiceImplTest.java (hunk 1 above)
Comment (lilymammadova) [reply src/test/java/org/liliya/hotelapp/service/ApartmentServiceImplTest.java]: fixed

File: src/test/java/org/liliya/hotelapp/service/ApartmentServiceImplTest.java (hunk 2 above)
Comment (lilymammadova) [reply src/test/java/org/liliya/hotelapp/service/ApartmentServiceImplTest.java]: fixed

File: src/test/java/org/liliya/hotelapp/ui/ConsoleTest.java (hunk 3 above)
Comment (lilymammadova) [reply src/test/java/org/liliya/hotelapp/ui/ConsoleTest.java]: removed

**Comment by lilymammadova** [comment]
@Alexmansar fixed all comments and about CI, the workflow already here : [https://github.com/lilymammadova/HotelApplication/blob/master/.github/workflows/main.yml](url) , I just already pushed it to master branch I thought that we need only checks in pull requests

**Review by Alexmansar — APPROVED** [review]
@lilymammadova Hi)) please, merge it) good job

**Pull request merged**